components/layout/Auth/AuthConfig.ts

```diff
@@ -0,0 +1,131 @@
+import { FocusEvent } from "react";
+import { FieldInfo, PageConfigType } from "@/types/AuthTypes";
+
+export enum FIELDTYPE {
+  EMAIL = "email",
+  NICKNAME = "nickname",
+  PASSWORD = "password",
+  PASSWORDCONFIRMATION = "passwordConfirmation",
+}
+
+export const fields: { [id: string]: FieldInfo } = {
```

Comment:
Record를 사용해주시면 더 좋을 것 같습니다.
```suggestion
export const fields: Record<FIELDTYPE, FieldInfo> = {
```